src/index.ts

```diff
@@ -80,14 +83,23 @@ async function run() {
       () => {
         return;
       },
-      async (args) =>
+      async (args) => {
+        if (failedState) {
+          console.log("Cannot send production message for a build, which did not produce an artifact.");
+          process.exit(1);
```

Comment:
thats not right. we don't have artifacts in prod at all. But we want to send a message for failed prod builds.

Reply:
we still have artifacts in the sence that a build is created on codemagic and the corresponding file is passed on. This indicates whether a successful build did occur. So far we do not send messages for unsuccesfull production builds. However I'm not entirely sure as to why. I guess we argued that this pipeline should only be triggered once the dev-pipeline ran succesfully, therefore it should be impossible to reach this case.

Reply:
@jkoenig134 should we simply add a negative Production message?

Reply:
I looked in the code. we already have one.

Reply:
I dont think we have. We have a negative build message - however a message about a failed prodction should be different from this.

When we call "jscm teams-production" - the message is produced by the following snippet:
![image](https://github.com/js-soft/codemagic-tools/assets/34318568/7b933f90-d62c-4c36-a978-e784a4bb90af)

Reply:
ok, in that case we defenitely need that message.